src/Tests/Tests/Indices/IndexManagement/CreateIndex/CreateIndexApiTests.cs

```diff
@@ -206,6 +206,9 @@ protected override void ExpectResponse(ICreateIndexResponse response)
 			response.Acknowledged.Should().BeTrue();
 			response.ShardsAcknowledged.Should().BeTrue();
 
+			if (Cluster.ClusterConfiguration.Version >= "6.8.0")
```

Comment:
index is returned in Elasticsearch 6.0.0, so this condition is not needed

Reply:
++ will remove